stylesheets/main.css

```diff
@@ -0,0 +1,252 @@
+:root {
+  --primary-color: #0077c5;
+  --white-text: #fff;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background: #eceef1;
+  height: 100vh;
+}
+
+.flex-h,
+.flex-v {
+  display: flex;
+}
+
+.flex-v {
+  flex-direction: column;
+}
+
+.align-center {
+  align-items: center;
+}
+
+.text-center {
+  text-align: center;
+}
+
+a {
+  text-decoration: none;
+  color: var(--primary-color);
+}
+
+.btn {
+  background: var(--primary-color);
+  border-radius: 0.3rem;
+  border: 1px solid #333;
+  color: var(--white-text);
+  padding: 0.5rem 1.4rem;
+}
+
+.m-r-1 {
+  margin-right: 1rem;
+}
+
+.m-r-2 {
+  margin-right: 2rem;
+}
+
+.m-t-1 {
+  margin-top: 1rem;
+}
+
+.m-t-2 {
+  margin-top: 2rem;
+}
+
+.m-t-3 {
+  margin-top: 3rem;
+}
+
+.m-t-4 {
+  margin-top: 4rem;
+}
+
+.m-b-1 {
+  margin-bottom: 1rem;
+}
+
+.m-b-2 {
+  margin-bottom: 2rem;
+}
+
+.m-b-3 {
+  margin-bottom: 3rem;
+}
+
+.m-b-4 {
+  margin-bottom: 4rem;
+}
+
+.p-1 {
+  padding: 1rem;
+}
+
+.p-2 {
+  padding: 2rem;
+}
+
+.fs-0-8 {
+  font-size: 0.7rem;
+}
+
+#main-header {
+  justify-content: flex-end;
+  padding: 0.6rem 1rem;
+  font-size: 0.8rem;
+  color: #777;
+}
+
+#main-header p {
+  margin-bottom: 0.1rem;
+}
+
+.btn i {
+  margin-right: 0.4rem;
+  font-size: 0.8rem;
+}
+
+#sign-in-button {
+  height: 35px;
+}
+
+#sign-in-button i {
+  margin-right: 0.3rem;
+}
+
+#logo div {
+  justify-content: center;
+  margin-top: 0.8rem;
+}
+
+#main-footer a {
+  margin-right: 1rem;
+  font-size: 0.8rem;
+  color: #333;
+  margin-bottom: 0.6rem;
+  display: inline-block;
+}
+
+#logo div a {
+  margin: 0 5px;
+  height: 100%;
+}
+
+.form-input span {
+  font-size: 0.8rem;
+}
+
+#logo span {
+  top: 5px;
+  margin: 0 4px;
+  color: #777;
+  font-size: 1.2rem;
+}
+
+#signup-form {
+  justify-content: space-between;
+}
+
+#signup-form form {
+  background: #fff;
+  height: 580px;
+  width: 23vw;
+  border: 1px solid #aaa;
+}
+
+#signup-form header {
+  color: #444;
+  font-weight: 300;
+  font-size: 1.6rem;
+}
+
+#signup-form header p {
+  font-size: 0.9rem;
+  color: #777;
+}
+
+#signup-form header a {
+  font-size: 0.9rem;
+}
+
+.form-input {
+  margin-bottom: 1.8rem;
+}
+
+.form-input label {
+  display: block;
+  margin: 0.6rem 0;
+  font-size: 1rem;
+  color: #444;
+  font-weight: 300;
+}
+
+.form-input input {
+  padding: 0.6rem 1.2rem;
+  width: 100%;
+  border-radius: 3px;
+  outline: none;
+  border: 1px solid #999;
+}
+
+.form-input input[type='tel'] {
+  margin-bottom: 0.5rem;
+}
+
+#pass {
+  position: relative;
+}
+
```

Comment:
Try to use less "ID" classes in css, because is a good practices to use classes instead, this is because classes are re-usable.

Reply:
Thanks, for your valuable feedback. Just letting you know we have applied your recommendations.